docs/tasks/administer_cluster_quotas.md

```diff
@@ -33,30 +33,30 @@ cluster.
 Write the manifest for the ClusterQueue. It should look similar to the following:
 
 ```yaml
-# cluster-total.yaml
+# cluster-queue.yaml
```

Comment:
I think cluster-total made the most sense in this case (Single ClusterQueue and single ResourceFlavor setup)

Reply:
I'd like to use the same names as much as possible in the documentation, so someone following it and copy-pasting the examples doesn't need to change much when jumping from one page to another